compiler/rustc_attr/src/builtin.rs

```diff
@@ -1036,21 +1036,21 @@ pub fn allow_internal_unstable<'a>(
     sess: &'a Session,
     attrs: &'a [Attribute],
 ) -> Option<impl Iterator<Item = Symbol> + 'a> {
-    allow_unstable(sess, attrs, sym::allow_internal_unstable)
+    Some(allow_unstable(sess, attrs, sym::allow_internal_unstable))
 }
 
 pub fn rustc_allow_const_fn_unstable<'a>(
     sess: &'a Session,
     attrs: &'a [Attribute],
 ) -> Option<impl Iterator<Item = Symbol> + 'a> {
-    allow_unstable(sess, attrs, sym::rustc_allow_const_fn_unstable)
+    Some(allow_unstable(sess, attrs, sym::rustc_allow_const_fn_unstable))
```

Comment:
Couldn't these two functions also not return `Option`?

Reply:
Maybe, but they are public functions which is why I don't want to touch them (there's risk of breaking other stuff easily)

Reply:
All the "other stuff" is also rustc and can be changed if broken.
These specific functions are also used only in couple of places.